pages/home.js

```diff
@@ -112,12 +120,38 @@ export async function getStaticProps({ locale }) {
     benefits = AEMbenefits.data.entities
   }
 
+  // Get list of top tasks
+  let topTasksReturned = await aemService.getFragment(
+    'alpha/components/top-tasks.json'
```

Comment:
base api variable has alpha in it

Reply:
I need to wait for your PR to be merged before I can remove the alpha or else the benefits inside of CardList.js crashes